src/app/app.component.ts

```diff
@@ -1,17 +1,24 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 
 import { AuthService } from './shared/auth.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html'
 })
-export class AppComponent implements OnInit{
+export class AppComponent implements OnInit, OnDestroy {
   isAuthorized: boolean;
+  _subscription: any;
```

Comment:
`this._subscription` is not of type `any`, but of type `Subscription` which can be imported from `rxjs/Subscription`